sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestRecordTests.java

```diff
@@ -90,4 +99,52 @@ public void cancelRecord() throws URISyntaxException, InterruptedException, Json
         assertThat(errorStatus).isNotNull();
         assertThat(errorStatus.toString()).isEqualTo(statusString);
     }
+
+    @Test(groups = { "fast" })
```

Comment:
Mixing Unit tests and fast tests is not a great idea. Now for the unit tests as well, you will initialize TestSuiteBase class for this particular class unnecessary. 
Can we keep them separate?

Reply:
where is the problem with initializing TestSuiteBase?

Reply:
I moved the tests into cosmosDiagnosticsTests file as part of the test is testing the serialization of the cosmos diagnostics